nav2_simple_commander/README.md

```diff
@@ -20,6 +20,7 @@ The methods provided by the basic navigator are shown below, with inputs and exp
 | goThroughPoses(poses)             | Requests the robot to drive through a set of poses (list of `PoseStamped`).|
 | goToPose(pose)                    | Requests the robot to drive to a pose (`PoseStamped`).                     |
 | followWaypoints(poses)            | Requests the robot to follow a set of waypoints (list of `PoseStamped`). This will execute the specific `TaskExecutor` at each pose.   |
+| followPath(path)                  | Requests the robot to follow a path from a starting to a goal `PoseStamped`, `nav_msgs/Path`.     |
```

Comment:
https://navigation.ros.org/commander_api/index.html Add to this API too!

Reply:
Updated the required changes at this PR https://github.com/ros-planning/navigation.ros.org/pull/272